modules/Book.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
+export class Book {
```

Comment:
- [ ] Please don't disable **eslint** because linter checks for those errors which can break your application. :blush: